app/controllers/admin/users_controller.rb

```diff
@@ -39,12 +39,20 @@ def index
         .order("users.full_name", "facilities.name", "users.device_created_at")
     end
 
-    @users =
-      if searching?
-        paginate(users.search_by_name_or_phone(search_query))
-      else
-        paginate(users)
+    respond_to do |format|
+      format.html do
+        @users =
+          if searching?
+            paginate(users.search_by_name_or_phone(search_query))
+          else
+            paginate(users)
+          end
+      end
+
+      format.json do
+        @users = users.search_by_name_or_teleconsultation_phone_number(search_query)
```

Comment:
Agree, I was avoiding a new one since there was a lot of churn on this page going on with the permissions stuff in parallel. I _think_ that has settled down now, added the controller in e1373e0. This uses the old system, I'll flag it to @vkrmis to take a look.